graphics/include/ignition/common/ColladaExporter.hh

```diff
@@ -51,7 +68,9 @@ namespace ignition
 
       public: void Export(const Mesh *_mesh,
           const std::string &_filename, bool _exportTextures,
-          const std::vector<math::Matrix4d> &_submeshToMatrix);
+          const std::vector<math::Matrix4d> &_submeshToMatrix,
```

Comment:
Adding a default-valued parameter to an existing function in a release will break ABI, asyou can see in the checker. 

https://build.osrfoundation.org/job/ignition_common-abichecker-any_to_any-ubuntu_auto-amd64/659/API_5fABI_20report/


In order to work around this and land it in ign-common4, I suggest introducing a second `Export` function with the new number of args and then calling that one from the old function.

```
void Export(const Mesh *_mesh,
          const std::string &_filename, bool _exportTextures,
          const std::vector<math::Matrix4d> &_submeshToMatrix,
          const std::vector<ColladaLight> &_lights)
{
    // Current impl goes here
}

public: void Export(const Mesh *_mesh,
          const std::string &_filename, bool _exportTextures,
          const std::vector<math::Matrix4d> &_submeshToMatrix)
{
    Export(_mesh, _filename, _exportTextures, _submeshToMatrix, std::vector<ColladaLight>());
}
```

Reply:
abi compatibility f6e2f92